milli/src/score_details.rs

```diff
@@ -256,6 +261,13 @@ impl ScoreDetails {
                     details_map.insert(vector, details);
                     order += 1;
                 }
+                ScoreDetails::Skipped => {
+                    details_map.insert(
+                        "skipped".to_string(),
+                        serde_json::Number::from_f64(0.).unwrap().into(),
```

Comment:
- the value of a detail must be an object (here it is a float)
- it must contain the "order" field.
- for "skipped", no more details are necessary (we don't need a 0).

So for skipped, the details should look:

```suggestion
                        serde_json::json!({"order": order})
```